language/vm/vm-runtime/src/interpreter.rs

```diff
@@ -914,10 +953,11 @@ impl<'txn> Interpreter<'txn> {
         current_frame: &Frame<'txn, FunctionRef<'txn>>,
     ) -> VMStatus {
         if err.is(StatusType::InvariantViolation) {
+            let state = self.get_internal_state(current_frame);
```

Comment:
no reason for this except it makes it easier to `println!()` it if we need it for debugging reasons